files/build_templates/sonic_debian_extension.j2

```diff
@@ -347,12 +348,31 @@ sudo https_proxy=$https_proxy LANG=C chroot $FILESYSTEM_ROOT pip3 install watchd
 sudo https_proxy=$https_proxy LANG=C chroot $FILESYSTEM_ROOT pip2 install futures==3.3.0
 
 {% if include_kubernetes == "y" %}
-# Copy kubelet service files
-# Keep it disabled until join, else it continuously restart and as well spew too many
-# non-required log lines wasting syslog resources.
+# kubelet service is controlled by ctrmgrd daemon.
 sudo LANG=C chroot $FILESYSTEM_ROOT systemctl disable kubelet.service
 {% endif %}
 
+# Copy remote container mangement files
+# File called from each container upon start/stop to record the state
+sudo mkdir -p ${FILESYSTEM_ROOT_USR_SHARE_SONIC_SCRIPTS}
+sudo mkdir -p ${FILESYSTEM_ROOT_ETC_SONIC}
+
+# Install remote Container mgmt package
+SONIC_CTRMGMT_WHEEL_NAME=$(basename {{sonic_ctrmgmt_py3_wheel_path}})
+sudo cp {{sonic_ctrmgmt_py3_wheel_path}} $FILESYSTEM_ROOT/$SONIC_CTRMGMT_WHEEL_NAME
+sudo https_proxy=$https_proxy LANG=C chroot $FILESYSTEM_ROOT pip3 install $SONIC_CTRMGMT_WHEEL_NAME
+sudo rm -rf $FILESYSTEM_ROOT/$SONIC_CTRMGMT_WHEEL_NAME
+
+sudo cp ${files_path}/container_state.py ${FILESYSTEM_ROOT_USR_SHARE_SONIC_SCRIPTS}/
+sudo chmod a+x ${FILESYSTEM_ROOT_USR_SHARE_SONIC_SCRIPTS}/container_state.py
+
+# Config file used by container mgmt scripts/service
+sudo cp ${files_path}/remote_ctr.config.json ${FILESYSTEM_ROOT_ETC_SONIC}/
+
+# Remote container management service files
+sudo cp ${files_path}/ctrmgrd.service ${FILESYSTEM_ROOT_USR_LIB_SYSTEMD_SYSTEM}/
+sudo LANG=C chroot $FILESYSTEM_ROOT systemctl enable ctrmgrd.service
```

Comment:
should we move all these lines inside include_kubernetes block?

Reply:
Good point. I will look into this.

Reply:
done